apps/indexer/templates/indexer/bits/profile_display.html

```diff
@@ -14,7 +14,7 @@
   <li> Show links to the wiki documentation: {{ profile_user.indexer.show_wiki_links|yesno }}
   <li> Use tabbed view while editing stories: {{ profile_user.indexer.use_tabs|yesno }}
 {% endif %}
-  <li> Country: <img src="{{ STATIC_URL }}/img/gcd/flags/{{ profile_user.indexer.country.code|lower }}.png" alt="{{ country }}" class="embedded_flag">
```

Comment:
You do have a working dev to test a change on the corresponding pages, right ?
Then we should do the full change here and replace
img src="{{ STATIC_URL }}/img/gcd/flags/{{ profile_user.indexer.country.code|lower }}.png" alt="{{ country }}" class="embedded_flag" (with the <> around)
by
{{ profile_user.indexer.country|show_country_flag }}
to have the code the generate the flag path only in one place
you might need to add
{% load credits %}
in the beginning of a template if not already there